packages/DynamicHyperlinkTransformer/src/DynamicHyperlinkTransformer.js

```diff
@@ -36,15 +36,24 @@ export default function DynamicHyperlinkTransformer({ onFetch }) {
               .then(response => {
                 const { error, name, term } = response;
 
-                const className = error ? "dynamic-hyperlink--invalid" : "dynamic-hyperlink--valid";
-                const linkText = error ? originalLinkUrl : name;
-                const errorText = error ? I18n.t(`dynamicHyperlinkTransformer.${error}`) : "";
-
-                dynamicHyperlink.innerHTML = linkText; // eslint-disable-line no-param-reassign
-                const typeOrErrorSpan = document.createElement("span");
-                typeOrErrorSpan.innerHTML = error ? `- ${errorText}` : term;
-                typeOrErrorSpan.className = className;
-                dynamicHyperlink.appendChild(typeOrErrorSpan);
+                dynamicHyperlink.innerHTML = ""; // eslint-disable-line no-param-reassign
+
+                const labelSpan = document.createElement("span");
+                labelSpan.className = "dynamic-hyperlink--label";
+                labelSpan.innerHTML = error ? originalLinkUrl : name;
```

Comment:
can be a case where the content on the tag requires sanitization? would be a way to inject malicious code via this link?

Reply:
This code only gets run on URLs with the 'data-dynamic-hyperlink' attribute, and links only get that attribute if they match a certain regex pattern defined in the back-end. So i think the back-end is essentially doing the sanitization with their regex.

Reply:
what about adding this, I think wouldn't hurt adding more security? https://github.com/apostrophecms/sanitize-html#readme

Reply:
`... It is well suited for cleaning up HTML fragments such as those created by CKEditor and...`.  Projects is already doing some stripping of fragments added by CKEditor in the back-end onSave.

Also, I am just putting the original content back into the link... I don't see how not sanitizing that can be dangerous?

Reply:
i'll follow up with r&d in another ticket to see if this is necessary. for now please just review the styling changes here.